src/services/rhsm/rhsmSchemas.js

```diff
@@ -108,6 +108,42 @@ const guestsResponseSchema = Joi.object().keys({
   meta: guestsMetaSchema.default({})
 });
 
+/**
+ * Hosts response meta field.
+ *
+ * @type {*} Joi schema
+ */
+const hostsMetaSchema = metaResponseSchema;
+
+/**
+ * Hosts response item.
+ *
+ * @type {*} Joi schema
+ */
+const hostsItem = Joi.object({
+  cloud_provider: Joi.string().optional().allow(null),
+  display_name: Joi.string().optional().allow(null),
+  hardware_type: Joi.string().optional().allow(null),
+  inventory_id: Joi.string().optional().allow(null),
+  measurement_type: Joi.string().optional().allow(null),
```

Comment:
required: the old hosts response flips `cloud_provider` `hardware_type` and `measurement_type` to all caps in the response... make them `Joi.string().lowercase().optional().allow(null)`